### DISCUSSION TITLE
Can we add custom feedback button?

### DISCUSSION THREAD

**Discussion by furkankaradas** [flyerhq/flutter_chat_ui#196] [Ideas & Requests] [answered]
Hi, 
It is an excellent library, thank you for everything. We want to add a feedback mechanism for Conversational AI, and we need to put buttons (range 1-5) in the text bubble and get this. Is it support this, ​or can we do this with custom types?

**Comment by demchenkoalex** [comment]
You mean like quick responses above the text input?

**Reply by furkankaradas** [reply]
Yes, it is.

**Reply by demchenkoalex** [reply]
https://github.com/flyerhq/flutter_chat_ui/discussions/188 I guess this is the same right?

**Reply by furkankaradas** [reply]
#188 It is like that "Was this answer helpful" "Yes" or "No" buttons in this picture.

**Reply by demchenkoalex** [reply]
oh, we didn't plan something like that. I guess we can think about it in the future, but it will be very distant future, having things we still need to do. You can achieve it by using CustomMessage now, you can replicate TextMessage and add buttons below it.

**Reply by furkankaradas** [reply]
Thank you for everything.

**Comment by demchenkoalex** [comment, marked as answer]
for v2 pointing to this https://github.com/flyerhq/flutter_chat_ui/discussions/188
